releases.md

```diff
@@ -57,3 +57,34 @@ Ign-(N+2)   | void bar();
 
 Check out [this table](https://github.com/ignitionrobotics/docs/blob/master/tools/versions.md)
 for a list of release and EOL dates for all versions of all libraries.
+
+### Supported platforms
```

Comment:
I'd rather not make a distinction between support provided by Open Robotics or by other parties, because end-users shouldn't care about that. This document is about the end-result: what's tested on CI, what's guaranteed to work? It doesn't matter who's putting the effort.

Reply:
I get the point. Somehow we are in a grey space where different groups (Debian, conda, etc) offer some level of CI (run unit test on every release, they have some own tests in place, etc) and they care (in some level) about bugs filed against their packages, etc.